automation/run-tests.sh

```diff
@@ -227,8 +227,16 @@ function modprobe_ovs {
     lsmod | grep -q ^openvswitch || modprobe openvswitch || { echo 1>&2 "Please run 'modprobe openvswitch' as root"; exit 1; }
 }
 
+function vdsm_tests {
+  git clone https://gerrit.ovirt.org/vdsm
+  cd vdsm
+  ./tests/network/functional/run-tests.sh --nmstate-source="$PROJECT_PATH"
```

Comment:
If you only need the shell script but nothing else from the repo, something like this is possible:

`curl https://raw.githubusercontent.com/oVirt/vdsm/master/tests/network/functional/run-tests.sh | bash -s - --nmstate-source="$PROJECT_PATH"`

 but since `run-test.sh` from vdsm does not use a main function that is called at the end, this execute only part of the script if there is a transmission error.

Reply:
Unfortunately we need whole repo because it is mounted into the container.